oe_theme.theme

```diff
@@ -1696,3 +1701,14 @@ function oe_theme_preprocess_patterns_overview_page(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_oe_theme_helper_site_navigation(&$variables) {
+  if ($variables['ecl_branding'] === 'core') {
+    // We shouldn't have site_name inside the navigation block
+    // with 'Core' ECL branding.
+    unset($variables['site_name']);
+  }
+}
```

Comment:
This should not be needed after this is merged https://github.com/openeuropa/oe_theme/pull/868/files#diff-7407f6e120cbb3baf41f389e45af0951cd1d39aff287c115d47d123d35f58311R49-R54

Reply:
Actually, 1250 should be rebased after this PR is merged.